infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -130,6 +130,8 @@
                     + "                                                                          "
                     + " -k <seqcache>, --sequencer-cache-size=<seqcache>                         "
                     + "               The size of the sequencer's cache. [default: 250000].\n    "
+                    + " -B <size> --batch-size=<size>"
+                    + "                 The batch size used to transfer data [default: 100].\n"
```

Comment:
Used to transfer data or bulk reads by the FastObjectLoader on sequencer failover.

Reply:
Changed. Also, trying to get rid of "bulk", "Batch" seems more appropriate.